in/install.sh.in

```diff
@@ -22,6 +22,9 @@ main() {
         in/unit/push-to-cloud.service.in > ${UNIT_DIR}/push-to-cloud.service \
         && sed -i '/User=ptc/d' ${UNIT_DIR}/push-to-cloud.service \
         && systemctl --user daemon-reload
+    
+    cp in/unit/ptc-vault.service ${UNIT_DIR}/ptc-vault.service \
```

Comment:
This file is actually redundant. Zac has implemented this here: https://github.com/broadinstitute/pkgbuilder-push-to-cloud/blob/master/push-to-cloud.spec#L33

Reply:
(I'll create a jira for me to update this)